app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabListItem.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray.inactivetabs
+
+import mozilla.components.browser.state.state.TabSessionState
+
+/**
+ * A data class used to display inactive tabs in the UI
+ *
+ * @param tabTitleText The tab's display text.
+ * @param displayUrl The tab's URL.
+ * @param tab The underlying [TabSessionState] used for any lambda callbacks.
+ */
+data class InactiveTabListItem(
```

Comment:
Don't think this is really necessary since we can get all the information from `TabSessionState` instead of wrapping it inside another class.